content/ai/concepts/generative-ai/concepts/images/images.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Images'
+Description: GenAI can generate new images from existing text prompts and images. Because of a random seed, the images generated are unique creations. GenAI uses diffusion models to create these new unique images.
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
+  - 'Transformers'
+Tags:
+  - 'AI'
```

Comment:
Large Language Model and Transformers would be good tags, but they would also need to be added to the [Tags file](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) in this PR.